src/hosted-buttons/types.js

```diff
@@ -55,4 +59,21 @@ export type RenderForm = ({|
   onClick: (data: mixed, actions: mixed) => void,
 |};
 
+type Request = {|
+  url: string,
+  headers: {|
+    [key: string]: string,
+    Authorization?: string,
```

Comment:
I couldn't figure out a way to do this with flow:

<img width="755" alt="Screenshot 2024-02-22 at 14 56 52" src="https://github.com/paypal/paypal-checkout-components/assets/3824954/b6873df3-c681-4a4e-aee4-3f7061e74274">

so I settled for adding the optional `Authorization` property (even though we know it won't be in this object if undefined)